tests/api/api/test_feature_sets.py

```diff
@@ -491,3 +532,207 @@ def test_features_list(db: Session, client: TestClient) -> None:
     assert (
         features_response["features"][1]["feature_set_digest"]["metadata"]["tag"] == tag
     )
+
+
+# There will be fields added (uid for example), but we don't allow any other changes
+def _assert_diff_contains_only_additions(original_object, new_object):
+    diff = DeepDiff(original_object, new_object, ignore_order=True,)
+    diff.pop("dictionary_item_added")
+    assert diff == {}
+
+
+def test_feature_vector_create(db: Session, client: TestClient) -> None:
```

Comment:
This file is becoming too big, I suggest to make it so that under `tests/api/api` you'll have:
```
feature_store
    __init__.py
    base.py
    test_feature_sets.py
    test_feature_vectors.py
```
that way you can put the shared functions in the `base.py` but still test files won't be huge

Reply:
Yes. Good suggestion. The file was indeed becoming too large. Done.